app/steps/ui/signout/template.html

```diff
@@ -0,0 +1,18 @@
+{% extends "includes/layout.html" %}
+
+{% block page_title %}
+{{ content.title | safe}}
+{% endblock %}
```

Comment:
Following the work in Adam's story PRO-2839 the page title is now handled in the layout template so we won't need to use the `page_title` block here and it can be removed.